src/components/global/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styles from "./Dialog.module.scss";
+
+type TDialog = {
+  closeOnOutsideClick: boolean,
+  onRequestClose: () => void,
+  open: boolean,
+  props: HTMLElement
```

Comment:
```suggestion

```
@hubert-deriv the list of exact props that are expected in Dialog have to be added here instead of props.
when you put props here, you're telling TS that Dialog can accept prop attribute like this: <Dialog prop=...
Or does it really accept such a prop somewhere in the code?